view/frontend/web/js/view/payment/method-renderer/cardknox-apple-pay-method.js

```diff
@@ -84,13 +84,19 @@ define([
 
             cardknoxApplePay.init(this);
         },
+        isSupportedApplePay: function () {
+            if (!window.ApplePaySession || !ApplePaySession.canMakePayments()) {
+                return false
+            } else {
+                return true;
+            }
```

Comment:
Does this work?
```suggestion
            return window.ApplePaySession && ApplePaySession.canMakePayments();
```

Reply:
@mstein-ck , Yes, it is working, I tested it.